Source/WebKit/UIProcess/RemoteLayerTree/RemoteLayerTreeDrawingAreaProxy.h

```diff
@@ -62,7 +62,7 @@ class RemoteLayerTreeDrawingAreaProxy : public DrawingAreaProxy {
     TransactionID lastCommittedLayerTreeTransactionID() const { return m_webPageProxyProcessState.transactionIDForPendingCACommit; }
 
 #if ENABLE(THREADED_ANIMATION_RESOLUTION)
-    void animationEffectStackWasAdded(Ref<RemoteAcceleratedEffectStack> effects);
+    void animationEffectStackWasAdded(PlatformLayer*, Ref<RemoteAcceleratedEffectStack> effects);
```

Comment:
Should probably rename this to `animationEffectStackWasAddedToLayer` and invert the order of parameters to match the naming sequence. This would also need to be done on the other classes with similar methods. This could be done in a follow-up patch if you prefer as well, definitely not blocking landing this.

Reply:
I fixed this, no need to worry about it.